pkg/kubelet/volumes.go

```diff
@@ -77,12 +77,18 @@ func (kl *Kubelet) mountExternalVolumes(pod *api.BoundPod) (volumeMap, error) {
 	for i := range pod.Spec.Volumes {
 		volSpec := &pod.Spec.Volumes[i]
 
+		podRef, err := api.GetReference(pod)
```

Comment:
@thockin already said he will buy me a yard of beer in the future.  If I make @erictune my friend for life, I might get another yard when I make one of these meet-ups.

Should all blank SelfLinks not fail or would I make it specific to type (in this case, BoundPods)?  That seems like a trivial change in ref.go, unless I'm missing something.